src/test/java/paymelah/logic/commands/CommandTestUtil.java

```diff
@@ -141,4 +142,15 @@ public static void showPersonAtIndex(Model model, Index targetIndex) {
         assertEquals(1, model.getFilteredPersonList().size());
     }
 
+    /**
+     * Updates {@code model}'s filtered list to show only the persons with {@code debt} in the
```

Comment:
I followed the AB3 showPersonAtIndex (line 135) documentation. I'm not very familiar with when to use {@code}, should I use it only for class names?

Reply:
TBH I think AB3 itself is a bit inconsistent. New issue time? 👀

Reply:
That's in a code block because `targetIndex` is physically present in the code (as the second parameter of `showPersonAtIndex`). `debt` does not appear anywhere in your function.

Reply:
Ah I see, will change now